lsl/correlator/uvutils.py

```diff
@@ -156,117 +158,70 @@ def compute_uvw(antennas, HA=0.0, dec=34.070, freq=49.0e6, site=lwa1, include_au
+        uvw[k,:,:] = bl * freq.ravel() / speedOfLight
         
     uvw.shape = (Nbase,3)+freq.shape
     
     return uvw
-
-
-def compute_uv_track(antennas, dec=34.070, freq=49.0e6, site=lwa1):
-    """
-    Whereas compute_uvw provides the uvw coverage at a particular time, 
-    compute_uv_track provides the complete uv plane track for a long 
-    integration.  The output is a three dimensional numpy array with 
-    dimensions baselines, uv, and 512 points along the track ellipses.  
-    Unlike compute_uvw, however, only a single frequency (in Hz) can be 
-    specified.
-    
-    .. versionchanged:: 0.4.0
-        Switched over to passing in Antenna instances generated by the
-        :mod:`lsl.common.station` module instead of a list of stand ID
-        numbers.
-        
-    .. versionchanged:: 1.0.0
-        Added a keyword (site) to specify the station used for the 
-        observation.
-    
-    .. versionchanged:: 2.0.1
-        Added support for ephem.Angle and astropy.coordinates.Angle instances 
-        for dec.
-        Added support for astropy.coordinates.EarthLocation instances for site.
-    """
-    
-    N = len(antennas)
-    Nbase = N*(N-1)//2
-    uvTrack = numpy.zeros((Nbase,2,512))
-    
-    # Phase center coordinates
-    # Convert numbers to radians and, for HA, hours to degrees
-    if isinstance(dec, ephem.Angle):
-        dec2 = dec*1.0
-    elif isinstance(dec, AstroAngle):
-        dec2 = dec.radian
-    else:
-        dec2 = dec * numpy.pi/180
-    if isinstance(site, AstroEarthLocation):
-        lat2 = site.lat.radian
-    else:
-        lat2 = site.lat
-    
-    # Coordinate transformation matrices
-    trans1 = numpy.matrix([[0, -numpy.sin(lat2), numpy.cos(lat2)],
-                           [1,  0,               0],
-                           [0,  numpy.cos(lat2), numpy.sin(lat2)]])
-                    
-    count = 0
-    for i,j in get_baselines(antennas, indicies=True):
-        # Go from a east, north, up coordinate system to a celestial equation, 
-        # east, north celestial pole system
-        xyzPrime = antennas[i].stand - antennas[j].stand
-        xyz = trans1*numpy.matrix([[xyzPrime[0]],[xyzPrime[1]],[xyzPrime[2]]])
-        xyz = numpy.ravel(xyz)
-        
-        uRange = numpy.linspace(-numpy.sqrt(xyz[0]**2 + xyz[1]**2), numpy.sqrt(xyz[0]**2 + xyz[1]**2), num=256)
-        vRange1 = numpy.sqrt(xyz[0]**2 + xyz[1]**2 - uRange**2)*numpy.sin(dec2) + xyz[2]*numpy.cos(dec2)
-        vRange2 = -numpy.sqrt(xyz[0]**2 + xyz[1]**2 - uRange**2)*numpy.sin(dec2) + xyz[2]*numpy.cos(dec2)
-        
-        uvTrack[count,0,0:256] = uRange * freq / numpy.array(speedOfLight)
-        uvTrack[count,1,0:256] = vRange1 * freq / numpy.array(speedOfLight)
-        uvTrack[count,0,256:512] = uRange[::-1] * freq / numpy.array(speedOfLight)
-        uvTrack[count,1,256:512] = vRange2[::-1] * freq / numpy.array(speedOfLight)
-        count = count + 1
-        
-    return uvTrack
```

Comment:
If we switch things back for `compute_uvw` we might as well as this back in.  We'll also need to fix that notebook where this was used but dropped.

Reply:
Reverted.